### DISCUSSION TITLE
Any way to get the center coordinates of all areas?

### DISCUSSION THREAD

**Discussion by gpanciera** [img-mapper/img-mapper#41] [Q&A] [answered]
Is there any way to get the center coordinates (or scaled coordinates) from all areas and say, for instance, iterate through them? I have some other interface elements I'd like to manipulate based on the center coordinates of all areas.

**Comment by NishargShah** [comment, marked as answer]
do you need something like this?

https://github.com/img-mapper/react-img-mapper/blob/204413abebcb85086c18f81f43d2c642e0370518/src/ImageMapper.tsx#L244-L250

**Reply by gpanciera** [reply]
Thanks, that does do the trick—was just thinking it would be cleaner if I could pull the numbers from the ImageMapper component. Appreciate the help
